Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+LABEL maintainer="vlados.soc@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .

```

Comment:
Copying the entire context with `COPY . .` can lead to larger image sizes if there are unnecessary files in the build context. Ensure that your `.dockerignore` file is correctly configured to exclude non-essential files.